.github/labels-aws.yml

```diff
@@ -74,35 +95,38 @@
   description: Amazon Elastic Kubernetes Service
   name: aws:eks
 - color: ff9900
-  description: AWS ElastiCache
+  description: Amazon ElastiCache
   name: aws:elasticache
 - color: ff9900
-  description: ""
+  description: AWS Elastic Beanstalk
   name: aws:elasticbeanstalk
 - color: ff9900
-  description: AWS Elasticsearch
-  name: aws:elasticsearch
```

Comment:
Thanks for identifying the duplicate! Looks good!
Just as an FYI: The label sync only ensures that these labels are set, it does not ensure that non-existing labels are removed (because in the repos we might want to have some unmanaged labels).
This is why I:
- Migrated all issues with the label `aws:elasticsearch` to the label `aws:es`.
- Manually removed the label.

Reply:
Thanks. I will do the same with `aws:msk` -> `aws:kafka`